content/practices/hiding-semantics/hiding-semantics-practice.html

```diff
@@ -63,7 +63,7 @@ <h2>Effects of Role <code>presentation</code></h2>
           <li>The element’s implied ARIA role and any ARIA states and properties associated with that role are hidden from assistive technologies.</li>
           <li>
             Text contained by the element, i.e., inner text, as well as inner text of all its descendant elements remains visible to assistive technologies.
-            Of course, text is explicitly hidden, e.g., styled with <code>display: none</code> or has <code>aria-hidden="true"</code> is not visible to assistive technologies.
+            Of course, text which is explicitly hidden, e.g., styled with <code>display: none</code> or has <code>aria-hidden="true"</code> is not visible to assistive technologies.
```

Comment:
```suggestion
            Of course, text that is explicitly hidden, e.g., styled with <code>display: none</code> or has <code>aria-hidden="true"</code>, is not visible to assistive technologies.
```

Reply:
Alright, your grammar knowledge is better than mine - I never bothered to understand the which/that difference.